client/src/assets/icons/spacer.tsx

```diff
@@ -10,7 +10,7 @@ function Spacer(props: SpacerPropTypes): JSX.Element {
 
   return (
     <>
-      <span className='sr-only'>{t('icons.spacer')}</span>
+      <span className='sr-only' aria-hidden='true' />
       <svg
         className='tick'
```

Comment:
Applying the suggestion may break the format, can you clean the `<></>` because it becomes useless without multiple children in the JSX.Component


```suggestion
      <svg
        className='tick'
        aria-hidden='true' 
```

Reply:
Yes, this is what needs to be done. There is no reason to have that `span` there at all.

Reply:
I clicked on "Commit suggestion" and Lint job failed, I did something wrong?

Reply:
Nope, the code wasn't formatted for prettier. If you opened your gitpod/local environment and run `git pull` to make sure that the changes are in your environment, you can go to the changed file, and format it. (press `ctrl + a`, then `alt + shift + f`. It will format it for you 👍 ).